python/src/aiconfig/editor/server/server.py

```diff
@@ -19,7 +19,7 @@
 )
 from aiconfig.model_parser import InferenceOptions
 from flask import Flask, request
-from flask_cors import CORS
+from flask_cors import CORS # TODO: add this to requirements.txt
```

Comment:
Please go ahead and add it in any diff

Reply:
Isn't it already in requirements.txt?

Fwiw, we may actually be able to remove it. My hunch is:
- prod implementations, client can actually send api requests to /api instead of needing to explicitly specify localhost:8080 and this should work for multiple server instances
- dev implementation, client may be able to just proxy to the right port (https://create-react-app.dev/docs/proxying-api-requests-in-development/) but TBD if that actually does avoid CORS issues

Reply:
re: ports and CORS, @rholinshead I think you understand this the best, but I'm not sure we need any special logic for prod. Since it just serves static files at the user-specified port, doesn't the user just need to start another instance on another port and then navigate there?

Reply:
Ya, this is just more of a note for the client side since we'll need to change it from the current implementation hard-coded to port 8080

Reply:
Where is it hard-coded to 8080? Isn't that just for dev? For prod, isn't it just wherever the user navigates in browser?

Reply:
https://github.com/lastmile-ai/aiconfig/blob/main/python/src/aiconfig/editor/client/src/utils/api.ts#L4 right now, not just for dev. This was from before the discussion on the weekend. Should be easy enough to update today